### PR TITLE
Add topolvm_volumegroup_size_bytes metric

### LINKED ISSUE #248: Output VG size to metrics
## What

TopoLVM outputs only `topolvm_volumegroup_available_bytes` metrics.
Add `topolvm_volumegroup_size_bytes` metrics to monitoring.

## How

TBD

## Checklist

- [ ] Finish implementation of the issue
- [ ] Test all functions
- [ ] Have enough logs to trace activities
- [ ] Notify developers of necessary actions

### PR DESCRIPTION
**Pull request by bells17**
resolve #248

### CODE REVIEW

File: lvmd/proto/lvmd.proto [hunk 1]
```diff
@@ -76,7 +76,8 @@ message WatchResponse {
 
 message WatchItem {
     uint64 free_bytes = 1;  // Free space of the volume group in bytes.
-    string device_class = 2;
+    uint64 size_bytes = 2;  // Size of the volume group in bytes.
+    string device_class = 3;
```
Comment (toshipp) [inline lvmd/proto/lvmd.proto:80]: Adding a new field between old fields breaks backward compatibility.
The new field should be placed last, IMO.

**Review by toshipp — CHANGES_REQUESTED** [review]

File: lvmd/proto/lvmd.proto (hunk 1 above)
Comment (ymmt2005) [reply lvmd/proto/lvmd.proto:80]: Right.  Protocol buffer keeps backward-compatibility
by not changing the index of fields.

**Comment by satoru-takeuchi** [comment]
No additional comment. Please reflect the comments by @toshipp and @ymmt2005 .

File: lvmd/proto/lvmd.proto (hunk 1 above)
Comment (bells17) [reply lvmd/proto/lvmd.proto]: Sorry... I fixed it.

**Comment by bells17** [comment]
@toshipp @satoru-takeuchi Thank you for your review. I fixed the point of your comment.
Would you check this PR again?

**Review by satoru-takeuchi — APPROVED** [review]

**Review by toshipp — APPROVED** [review]
LGTM

**Pull request merged**